### DISCUSSION TITLE
Problemas con el multiselect

### DISCUSSION THREAD

**Discussion by LissethGeorginaFuentesFigueroa** [primefaces/community#654] [PrimeReact Blocks]
Estoy haciendo un modal de registro de descuentos con boostrapt, al usar el multselect este se muestra como si estuviera detrás del modal 
![image](https://github.com/primefaces/community/assets/105747977/8bdb0f53-4747-48a7-aa2e-7c605f9890f0)

No entiendo por qué muestra de esta manera el input

**Comment by nitrogenous** [comment]
Hello there, 
Thank you for your patience. I would love to assist you if you can share a stackblitz or codepen with me. 
Warmest regards
